### ISSUE TITLE
[inputs.sensors] throwing errors when run as a VM

### ISSUE THREAD

**Issue by stevemeuse** [Sabuto/hassio-repo#12]
First, thanks for making this add-on!

I'm running the telegraf add-on on HassOS as a VM on proxmox. I'm getting these log messags:

```
2023-03-23T00:41:40Z E! [inputs.sensors] Error in plugin: failed to run command "/usr/bin/sensors -A -u": exit status 1 - 
```

I didn't see any option to disable that input.  No major issue, just wasting log space, would be cool if there was a way to disable. 

-Steve

**Comment by sabuto** [comment]
Yeah I seem to be getting this now on my proxmox host, I think it has something to do specifically with hassos I will look into it
